spec/models/section_spec.rb

```diff
@@ -104,9 +106,18 @@ def key_classes_for(hash)
     )
   }
 
-  describe '.find' do
-    let(:manual) { double(:manual, slug: 'manual-slug') }
+  before do
+    allow(SlugGenerator).to receive(:new).with(prefix: manual_slug).and_return(slug_generator)
+  end
+
+  describe '#initialize' do
+    it 'instantiates a slug generator using the manual slug' do
+      expect(SlugGenerator).to receive(:new).with(prefix: manual_slug)
+      subject
+    end
```

Comment:
I've removed this test. I think you're right that it's probably good enough that we're stubbing the method to inject a mock.